environment-wsl2.yaml

```diff
@@ -4,8 +4,9 @@ channels:
   - defaults
 dependencies:
   - python=3.10
-  - pip=22.2.2
-  - cudatoolkit=11.3
-  - pytorch=1.12.1
-  - torchvision=0.13.1
-  - numpy=1.23.1
\ No newline at end of file
+  - pip=23.0
+  - cudatoolkit=11.8
+  - pytorch=2.0
+  - torchvision=0.15
+  - numpy=1.23
```

Comment:
@vladmandic Didn't you say Torch 2.0 requires Numpy 1.24+ instead of 1.23?

Reply:
Beta did, but they relaxes it for GA.
And some other dependencies require 1.23, so it's not that clean to use 1.24 just yet. Thus latest from 1.23.